eosetl/cli/export_all.py

```diff
@@ -0,0 +1,101 @@
+# MIT License
+#
+# Copyright (c) 2018 Evgeny Medvedev, Omidiora Samuel evge.medvedev@gmail.com, samparsky@gmail.com
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+
+import click
+import re
+
+from datetime import datetime, timedelta
+from eosetl.enumeration.chain import Chain
+from eosetl.jobs.export_all import export_all as do_export_all
+from eosetl.service.btc_block_range_service import BtcBlockRangeService
+from eosetl.rpc.eos_rpc import EosRpc
+from blockchainetl.thread_local_proxy import ThreadLocalProxy
+
+
+def is_date_range(start, end):
+    """Checks for YYYY-MM-DD date format."""
+    return bool(re.match('^2[0-9]{3}-[0-9]{2}-[0-9]{2}$', start) and
+                re.match('^2[0-9]{3}-[0-9]{2}-[0-9]{2}$', end))
+
+
+def is_block_range(start, end):
+    """Checks for a valid block number."""
+    return (start.isdigit() and 0 <= int(start) <= 99999999
+            and end.isdigit() and 0 <= int(end) <= 99999999)
+
+
+def get_partitions(start, end, partition_batch_size, provider_uri):
+    """Yield partitions based on input data type."""
+    if is_date_range(start, end):
+        start_date = datetime.strptime(start, '%Y-%m-%d').date()
+        end_date = datetime.strptime(end, '%Y-%m-%d').date()
+
+        day = timedelta(days=1)
+
+        btc_service = BtcBlockRangeService(
```

Comment:
Check everywhere btc prefix should be replaced with eos prefix.

Reply:
done